lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1015,6 +1029,20 @@ func (w *WalletKit) prepareSweepParams(in *BumpFeeRequest,
 			"pending inputs: %w", err)
 	}
 
+	if in.TargetConf != 0 {
```

Comment:
Should we also check if we don't already have a fee rate (`feerate.IsNone()`)? Otherwise we'd overwrite the sat/vByte value if that's specified (probably should add a check that not both is set as well).

Reply:
moved this check to `validateBumpFeeRequest`